ui/packages/statement/src/components/StatementsTable.tsx

```diff
@@ -1,111 +1,112 @@
       key: 'sum_latency',
-      sorter: (a: StatementOverview, b: StatementOverview) =>
-        a.sum_latency! - b.sum_latency!,
-      render: (value) => (
+      fieldName: 'sum_latency',
+      minWidth: 170,
+      isSorted: true,
+      isSortedDescending: true,
+      onColumnClick: onColumnClick,
+      onRender: (rec) => (
         <TextWithHorizontalBar
-          text={getValueFormat('ns')(value, 2, null)}
-          factor={value / maxSumLatency}
-          color={BLUE_COLOR}
+          text={getValueFormat('ns')(rec.sum_latency, 1, null)}
+          normalVal={rec.sum_latency / maxMins.maxSumLatency}
         />
       ),
     },
     {
-      title: t('statement.common.exec_count'),
-      dataIndex: 'exec_count',
-      key: 'exec_count',
-      sorter: (a: StatementOverview, b: StatementOverview) =>
-        a.exec_count! - b.exec_count!,
-      render: (value) => (
-        <TextWithHorizontalBar
-          text={getValueFormat('short')(value, 0, 0)}
-          factor={value / maxExecCount}
-          color={BLUE_COLOR}
-        />
-      ),
+      name: t('statement.common.avg_latency'),
+      key: 'avg_latency',
+      fieldName: 'avg_latency',
+      minWidth: 170,
+      onColumnClick: onColumnClick,
+      onRender: (rec) => {
+        const tooltipContent = `
+AVG: ${getValueFormat('ns')(rec.avg_latency, 1, null)}
+MIN: ${getValueFormat('ns')(rec.avg_latency * 0.5, 1, null)}
+MAX: ${getValueFormat('ns')(rec.avg_latency * 1.2, 1, null)}`
+        return (
+          <TextWithHorizontalBar
+            tooltip={<pre>{tooltipContent.trim()}</pre>}
+            text={getValueFormat('ns')(rec.avg_latency, 1, null)}
+            normalVal={rec.avg_latency / maxMins.maxAvgLatency}
+            maxVal={(rec.avg_latency / maxMins.maxAvgLatency) * 1.2}
+            minVal={(rec.avg_latency / maxMins.maxAvgLatency) * 0.5}
+          />
+        )
+      },
     },
     {
-      title: t('statement.common.avg_latency'),
-      dataIndex: 'avg_latency',
-      key: 'avg_latency',
-      sorter: (a: StatementOverview, b: StatementOverview) =>
-        a.avg_latency! - b.avg_latency!,
-      render: (value) => (
+      name: t('statement.common.exec_count'),
+      key: 'exec_count',
+      fieldName: 'exec_count',
+      minWidth: 170,
+      onColumnClick: onColumnClick,
+      onRender: (rec) => (
         <TextWithHorizontalBar
-          text={getValueFormat('ns')(value, 2, null)}
-          factor={value / maxAvgLatency}
-          color={BLUE_COLOR}
+          text={getValueFormat('short')(rec.exec_count, 0, 0)}
+          normalVal={rec.exec_count / maxMins.maxExecCount}
         />
       ),
     },
     {
-      title: t('statement.common.avg_mem'),
-      dataIndex: 'avg_mem',
+      name: t('statement.common.avg_mem'),
       key: 'avg_mem',
-      sorter: (a: StatementOverview, b: StatementOverview) =>
-        a.avg_mem! - b.avg_mem!,
-      render: (value) => (
+      fieldName: 'avg_mem',
+      minWidth: 170,
```

Comment:
according to my experience, also better to set a maxWidth for each fields, so that the auto layout (when user initially loads the page) will work. Auto layout will arrange each columns according to maxWidth in ratio.